presto-main/src/main/java/com/facebook/presto/sql/planner/plan/SemiJoinNode.java

```diff
@@ -140,7 +140,7 @@ public Symbol getSemiJoinOutput()
     }
 
     @Override
-    public PlanNode replaceChildren(List<PlanNode> newChildren)
+    public SemiJoinNode replaceChildren(List<PlanNode> newChildren)
```

Comment:
Unrelated change. It's not consistent with `JoinNode::replaceChildren`.

Reply:
It's related, used in `ReplicateSemiJoinUnderDelete`